jef/streetmerchant
# feat: add redis

### Description

Adds Redis as a way to keep track of stock. Redis connection string can be passed in to meet many different configurations.

### Testing


1. Startup redis (I used docker) with the default port
2. Setup your .env file to set the url (REDIS_URL="redis://localhost:6379/1")
3. Run the test command and verify that redis has a key in it.

### New dependencies
`redis`
`@types/redis`
